react-rendering-lists/my-app/src/PokemonList.tsx

```diff
@@ -0,0 +1,22 @@
+type Pokemon = {
+  number: string;
+  name: string;
+};
+
+type Pokedex = Pokemon[];
+
+const pokedex: Pokedex = [
```

Comment:
According to the exercise, these need to be declared in App. The reason is to get you some practice passing props. You can export the type and import it in App.